src/components/FeaturedProducts/FeaturedProducts.js

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import PropTypes from "prop-types"
+import styles from './FeaturedProducts.module.scss'
+import CarouselGrid from '../CarouselGrid/CarouselGrid';
+
+export default class FeaturedProducts extends React.Component {
+
+
+    constructor(props) {
+        super(props)
+
+        //   console.log("FeaturedProducts constructor******************",this.props.featuredProducts)  
+        var featuredProducts = this.props.featuredProducts.map
```

Comment:
Lets use const, let instead, is safer! :)

Reply:
my mistake. :). corrected in delivery 4